neuralpp/test/quick_tests/symoblic/simplication_test.py

```diff
@@ -0,0 +1,54 @@
+"""
+Tests of SymPy's simplication functions.
+https://docs.sympy.org/latest/tutorial/simplification.html
+"""
+from sympy import *
+
+
+def test_simplify() -> None:
+    """
+    One can just use sympy.simplify().
+    It's push-button but has 2 caveats:
```

Comment:
What does "push-down" mean?

Reply:
Ah, now I understand. I think many people may get confused like I did, so can you please add a short explanation ("automatic and general", something like that)?

Reply:
I replaced "push-button" with "general", does that make sense?